src/utils/wallet.ts

```diff
@@ -515,42 +528,47 @@ const fetchEngineTokensData = async (username: string, hivePrice: number, vsCurr
             precision: item.precision,
             actions,
             volume24h: volume24h,
-            extraDataPairs: [{
-              dataKey: 'staked',
-              value: item.stake !== 0 ? `${item.stake}` : '0.00'
-            }, {
-              dataKey: 'delegations_in',
-              value: item.delegationsIn !== 0 ? `${item.delegationsIn}` : '0.00'
-            }, {
-              dataKey: 'delegations_out',
-              value: item.delegationsOut !== 0 ? `${item.delegationsOut}` : '0.00'
-            }]
+            extraDataPairs: [
+              {
+                dataKey: 'staked',
+                value: item.stake !== 0 ? `${item.stake}` : '0.00',
+              },
+              {
+                dataKey: 'delegations_in',
+                value: item.delegationsIn !== 0 ? `${item.delegationsIn}` : '0.00',
+              },
+              {
+                dataKey: 'delegations_out',
+                value: item.delegationsOut !== 0 ? `${item.delegationsOut}` : '0.00',
+              },
+            ],
           };
         }
       });
     }
   } catch (err) {
-    console.warn("failed to get engine tokens data", err);
+    console.warn('failed to get engine tokens data', err);
   }
 
   return engineCoinData;
+};
 
-
```

Comment:
try not to include all lint results in PR, I do have plans to an app wide some time soon, the ts rules needs to be updated before we can properly do that... for now only hand pick linted files that you change in a particular PR. Another tip regarding lint.

I personally usually skip pre-commit hooks and lint manually before PR as some times lint can also break the app, not doing auto commit with lint gives me a chance to make sure lint did not break anything in app